packages/astro/src/integration/vite-plugin-astro-config.ts

```diff
@@ -30,10 +30,10 @@ export function vitePluginAstroConfig(astroConfig: AstroConfig): VitePlugin {
     load(id) {
       if (id === resolvedVirtualModuleId) {
         return `
-          export const astroConfig = ${JSON.stringify(astroConfig)};
```

Comment:
We actually are not using this anywhere in the astro codebase, and this is internal, so it should be safe too remove the export and just use the needed property